package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-no-only-tests": "^2.3.0",
     "eslint-plugin-react": "^7.12.3",
     "solhint": "^1.5.0",
-    "solidity-coverage": "^0.6.7",
+    "solidity-coverage": "^0.7.1",
```

Comment:
Is this update intended?

Reply:
That is in a different PR I am based off, but its not working so I will revert.